### PR TITLE
Replace trilead with jsch

### PR DESCRIPTION
**Pull request by mskdeepak-oracle**
- Removed the trilead directory and replaced it with jsch

### CODE REVIEW

File: jsch/pom.xml [hunk 1]
```diff
@@ -73,12 +72,13 @@
         <connection>scm:git:git@github.com:javaee/repackaged.git</connection>
         <developerConnection>scm:git:git@github.com:javaee/repackaged.git</developerConnection>
         <url>https://github.com/javaee/repackaged.git</url>
-    </scm>
+      <tag>HEAD</tag>
```
Comment (yaminikb) [inline jsch/pom.xml]: Why is the tag needed?

Comment (mskdeepak-oracle) [reply jsch/pom.xml]: Don't think it's needed. The default value of this is HEAD.

**Comment by romain-grecourt** [comment]
Why replace trilhead ? Just add a new folder for jsch.

**Comment by mskdeepak-oracle** [comment]
We will no longer be using trilead in glassfish. So, why do we still want the trilead folder here?

**Comment by romain-grecourt** [comment]
This repo isn't GF but just a placeholder for repackaged dependencies that are released and integrated into GF.

Even if removing Trilead is the right thing to do, doing a replace isn't right.
Instead do the remove and add in separate commits.

File: jsch/release.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+#
+# Copyright (c) 2010-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# The contents of this file are subject to the terms of either the GNU
+# General Public License Version 2 only ("GPL") or the Common Development
+# and Distribution License("CDDL") (collectively, the "License").  You
+# may not use this file except in compliance with the License.  You can
+# obtain a copy of the License at
+# https://oss.oracle.com/licenses/CDDL+GPL-1.1
+# or packager/legal/LICENSE.txt.  See the License for the specific
```
Comment (yaminikb) [inline jsch/release.sh]: Please fix the path to LICENSE.txt (line #13 and #17)

**Review by yaminikb — CHANGES_REQUESTED** [review]

**Review by yaminikb — APPROVED** [review]

**Pull request merged**